moz_security/sandboxes/heka/analysis/moz_security_auth_lastx.lua

```diff
@@ -49,16 +60,30 @@ default_email = "foxsec-dump+OutOfHours@mozilla.com" -- optional, enable email a
 -- drift_email = "captainkirk@mozilla.com" -- optional drift message notification
 -- acceptable_message_drift = 600 -- optional, defaults to 600 seconds if not specified
 
-expireolderthan = 864000 -- optional, tracked entries older than value are removed, defaults to 864000
-authhost_field = "Hostname" -- required, field to extract authenticating host from (destination host)
-user_field = "Fields[user]" -- required, field to extract username from
-track_fields = { "Fields[ssh_remote_ipaddr]" } -- required, fields to extract tracking data from
--- track_fields = { "Fields[ssh_remote_ipaddr_city]", "Fields[ssh_remote_ipaddr_country]" }
-
--- The geocity_field and geocountry_field values are optional, but if set and they are included
--- with the message and will be appended to the alert text as additional informational data
-geocity_field = "Fields[ssh_remote_ipaddr_city"] -- optional, field to extract geo city
-geocountry_field = "Fields[ssh_remote_ipaddr_country"] -- optional, field to extract geo country
+-- expireolderthan = 864000 -- optional, tracked entries older than value are removed, defaults to 864000
+-- lastx = 5 -- optional, track last X entries, defaults to 5
+
+event_fields = {
```

Comment:
Do we want to make this an array so we can control the order of evaluation?  The key name is this case is basically unused.

Reply:
The key name is used in the output right now (it's part of the subject to indicate the matched event type)